packages/next/pages/_document.js

```diff
@@ -26,14 +26,31 @@ export default class Document extends Component {
     }
   }
 
-  render () {
-    return <html amp={this.props.amphtml ? '' : null}>
-      <Head />
-      <body>
-        <Main />
-        <NextScript />
-      </body>
-    </html>
+  render() {
+    return (
+      <Html>
+        <Head />
+        <body>
+          <Main />
+          <NextScript />
+        </body>
+      </Html>
+    )
+  }
+}
+
+export class Html extends Component {
+  static contextTypes = {
+    _documentProps: PropTypes.any,
+  }
+
+  static propTypes = {
+    children: PropTypes.node.isRequired,
+  }
+
+  render() {
+    const { amphtml } = this.context._documentProps
+    return <html amp={amphtml ? '' : null}>{this.props.children}</html>
```

Comment:
probably you should spread the other props here (eg. in case you want to set the `lang` attribute on the `html` tag)

Reply:
woops I didn't see @Janpot's comment 😅